syntax/compile.go

```diff
@@ -258,7 +258,8 @@ func (pc ParseContext) compileDictPattern(b ast.Branch) rel.Pattern {
 				entryPtns = append(
 					entryPtns,
 					rel.NewDictPatternEntry(
-						rel.DotIdent,
+						// TODO: ???
```

Comment:
?

Reply:
I've updated the message. If you know the answer, I'd like to resolve this rather than leave it as a TODO.